Include/internal/pycore_runtime.h

```diff
@@ -229,6 +235,7 @@ typedef struct pyruntimestate {
     struct _fileutils_state fileutils;
     struct _faulthandler_runtime_state faulthandler;
     struct _tracemalloc_runtime_state tracemalloc;
+    struct _reftracer_runtime_state reftracer;
```

Comment:
Since it's called RefTracer, you may add an underscore for readability:

```suggestion
    struct _reftracer_runtime_state ref_tracer;
```